docker/presets/models/tfs/Dockerfile

```diff
@@ -24,7 +25,14 @@ COPY kaito/presets/workspace/inference/${MODEL_TYPE}/inference_api.py \
     /workspace/tfs/
 
 # 2. vLLM
-COPY kaito/presets/workspace/inference/vllm/inference_api.py /workspace/vllm/inference_api.py
+COPY kaito/presets/workspace/inference/vllm/inference_api.py \
+    kaito/presets/workspace/inference/vllm/multi-node-health-check.py \
+    /workspace/vllm/
+
+RUN VLLM_VERSION=$(grep 'vllm==' /workspace/requirements.txt | cut -d'=' -f3) && \
```

Comment:
I find it will give a prompt leaving you for decision to enable usage stats or not in 10s timeout at the first time you run "ray start xxxx".
> root@phi-4-mini-instruct-0:/workspace# /workspace/vllm/multi-node-serving.sh leader --ray_cluster_size=2 --ray_port=6379 --include-dashboard=True
Enable usage stats collection? This prompt will auto-proceed in 10 seconds to avoid blocking cluster startup.

add env  RAY_USAGE_STATS_ENABLED=1 to skip this waiting.

https://docs.ray.io/en/latest/cluster/usage-stats.html#how-does-it-work